v2/features/farms/FarmTableRowBodyTransactionControls.tsx

```diff
@@ -63,10 +63,11 @@ const FarmsTableRowBodyTransactionControls: FC<Props> = ({ jar }) => {
               storeAttribute="jarAllowance"
               chainName={jar.chain}
               visible={!userHasJarAllowance}
+              type="jar"
             />
             {userHasJarAllowance && (
               <div className="grid grid-cols-2 gap-3">
-                <DepositFlow jar={jar} balances={userTokenData} />
+                <DepositFlow jarOrBrinery={jar} balances={userTokenData} type="jar" />
```

Comment:
Suggestion, let's refer to the objects here as just `asset`. It's `jarOrBrinery` now but in the next iteration it will also need to include standalone farms and external assets so I know we'll need a refactor of this.